### PR TITLE
update docs links

### PR DESCRIPTION
**Pull request by PrajwalPrashanth**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ code .
 
 ## Contributing to Docs
 
-The documentation website of Jovian is hosted at [docs.jovain.ml](https://docs.jovian.ml/en/latest/)
+The documentation website of Jovian is hosted at [jovian.ml/docs](https://jovian.ml/docs)
```
Comment (siddhantujjain) [inline CONTRIBUTING.md]: You can use `https://jovian.ml/docs/`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ pip install -r requirements.txt
 
 To ensure that [VSCode](https://code.visualstudio.com/) and other code editors use the correct conda environment, run the following command to add the path of the `jovian-py-dev` env to your `.bashrc` file:
 
-*Ensure that you have activated `jovian-py-dev` the environment before running the following command* 
+_Ensure that you have activated `jovian-py-dev` the environment before running the following command_
```
Comment (rohitsanj) [inline CONTRIBUTING.md]: activated `jovian-py-dev` the environment --> 
activated `jovian-py-dev` environment 

I think the *the* can be dropped..

Comment (PrajwalPrashanth) [reply CONTRIBUTING.md]: Thanks will do.

File: CONTRIBUTING.md (hunk 1 above)
Comment (PrajwalPrashanth) [reply CONTRIBUTING.md]: done

**Pull request merged**